### PR TITLE
OSD-8736

### PR DESCRIPTION
**Pull request by robotmaxtron**
Boilerplate update && Routing NodeClockNotSynchronising to PagerDuty (plus small typo update).

### CODE REVIEW

File: pkg/controller/secret/secret_controller.go [hunk 1]
```diff
@@ -229,6 +229,10 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 		// https://issues.redhat.com/browse/OSD-3326
 		{Receiver: receiverPagerduty, Match: map[string]string{"cluster": "elasticsearch", "prometheus": "openshift-monitoring/k8s"}},
 
+		// Ensure NodeClockNotSynchronising is routed to SRE
+		// https://issues.redhat.com/browse/OSD-8736
+		{Receiver: receiverPagerduty, Match: map[string]string{"alertname": "NodeClockNotSynchronising", "prometheus": "openshift-monitoring/k8s"}},
```
Comment (cblecker) [inline pkg/controller/secret/secret_controller.go:234]: This won't change the severity of the alert.. it's still only going to come in as a warning. You may need some changes to create a receiver that actually manipulates the severity label per the card

**Comment by robotmaxtron** [comment]
Verified in stage, this alert will now reflect upstream's severity levels but the pagerdutyconfig will still set it an Error level so it pages out.

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**